www/app/page.tsx

```diff
@@ -289,11 +279,7 @@ export default function Home() {
   }
 
   return (
-    <main
-      className={`flex h-[100dvh] w-screen flex-col pb-[env(keyboard-inset-height)] text-sm lg:text-base overflow-hidden relative ${
-        isDarkMode ? 'dark' : ''
-      }`}
-    >
+    <main className="flex w-screen flex-col pb-[env(keyboard-inset-height)] text-sm lg:text-base overflow-hidden relative h-full">
       <Sidebar
```

Comment:
Can't open the left sidebar in mobile
<img width="377" alt="image" src="https://github.com/user-attachments/assets/522131a2-199f-463c-aa7d-2a423d747455">